native/CMakeLists.txt

```diff
@@ -596,6 +600,8 @@ elseif(ANDROID OR OHOS)
         cocos/platform/java/modules/CommonSystem.h
         cocos/platform/java/modules/SystemWindow.cpp
         cocos/platform/java/modules/SystemWindow.h
+        cocos/platform/java/modules/XRInterface.cpp
+        cocos/platform/java/modules/XRInterface.h
```

Comment:
The same as above.

Reply:
This is the implementation of XR interface class. XR macros are used inside the function, so that it can be compiled successfully without XR static library.